MAUI/ProgressBar/getting-started.md

```diff
@@ -0,0 +1,249 @@
+<progressBar:SfCircularProgressBar Progress="75"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+// Using linear progress bar. 
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar { Progress = 75 };
+this.Content = linearProgressBar;
+
+// Using circular progress bar.
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar { Progress = 75 };
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> By default, the value of progress should be specified between 0 and 100. To specify progress value between 0 and 1, specify the Minimum property to 0 and Maximum property to 1.
+
+Run the project, and check if you get following output to make sure that the project has been configured properly to add the progress bar.
+
+![.NET MAUI linear progress bar and circular progress bar](images/getting-started/progress-bar.png)
+
+## Enabling indeterminate state
+
+When the progress of a task cannot be shown determinately, you can enable the indeterminate state using the [IsIndeterminate]() property to know that any progress is happening in the background.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<!--Using linear progress bar-->
+<progressBar:SfLinearProgressBar IsIndeterminate="True"/>
+
+<!--Using circular progress bar-->
+<progressBar:SfCircularProgressBar IsIndeterminate="True"/>
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+// Using linear progress bar.
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar { IsIndeterminate = true };
+this.Content = linearProgressBar;
+
+// Using circular progress bar.
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar { IsIndeterminate = true };
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %} 
+
+## Enable segments
+
+To visualize the progress of a multiple sequential task, split the progress bar into the multiple segments by defining the `SegmentCount` property as demonstrated in the following code sample.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<!--Using linear progress bar-->
+
+<progressBar:SfLinearProgressBar SegmentCount="4" Progress="75"/>
+
+<!--Using circular progress bar-->
+
+<progressBar:SfCircularProgressBar SegmentCount="4" Progress="75"/>
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+// Using linear progress bar.
+
+SfLinearProgressBar linearProgressBar = new SfLinearProgressBar { Progress = 75, SegmentCount = 4 };
+this.Content = linearProgressBar;
+
+// Using circular progress bar.
+
+SfCircularProgressBar circularProgressBar = new SfCircularProgressBar { Progress = 75, SegmentCount = 4 };
+this.Content = circularProgressBar;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI linear progress bar and circular progress bar visualized with multiple sequential task](images/getting-started/segment.png)
+
+## Apply colors
+
+You can customize the color of the progress indicator and track by defining the `ProgressFill` and `TrackFill` properties, respectively.
```

Comment:
Customize the color of the progress indicator and track by defining the `ProgressFill` and `TrackFill` properties, respectively.

Reply:
Done